allenact/setup.py

```diff
@@ -3,20 +3,61 @@
 
 from setuptools import find_packages, setup
 
+
+def parse_req_file(fname, start=None):
+    reqs = {} if start is None else start
+    cline = None
+    with open(fname, "r") as f:
+        for line in f.readlines():
+            line = line[:-1]
```

Comment:
Can we make this `line = line[:-1].strip()` for defensiveness?

Reply:
👍